tendermint/vue
# fix: Disabled create CRUD button if not logged in

## Description & Context

If we do not have authorization, we should not allow the creation and disable the button

## How To Test

1. Open the Data section;
2. The "Create" button should be disabled;
3. Connect with Keplr;
4. The button must be active;

## Linked issue

Closes #203
